docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -130,3 +141,252 @@ You will also notice that the `docker-compose.yaml` file also has the following
 ```
 
 they are necessary to allow delve to attach to the running process and debug it and should not be used in production environments.
+
```

Comment:
The following additions don't really fit with the title IMHO. 

One thing I'm constantly looking for in our docs is how to extend the create-plugin configs and I can never find them. I wouldn't expect to find them in the "setting up dev env" page.

Reply:
Could you suggest an alternative - either to rename the setting up dev env page or where to have the extend config page?

Reply:
> The following additions don't really fit with the title IMHO.
> 
> One thing I'm constantly looking for in our docs is how to extend the create-plugin configs and I can never find them. I wouldn't expect to find them in the "setting up dev env" page.

I have captured this issue [here](https://github.com/grafana/plugin-tools/issues/1013) so it doesn't delay the re-org.